README.md

```diff
@@ -18,3 +18,42 @@ dotnet tool install --global Uno.PackageDiff
 ```
 generatepkgdiff --base=Uno.UI --other=C:\temp\Uno.UI.1.43.0-PullRequest0621.917.nupkg --outfile=diff.md
 ```
+
+The tool returns a non-zero value when differences are found, otherwise zero.
+
+## How to provide an ignore set
```

Comment:
How do you provide this file to the CI ?

Reply:
It would be a file committed in the repo. I'll a usage sample once it gets stabilized.